prober/prober.go

```diff
@@ -117,7 +129,14 @@ func (p *Prober) Run(name string, interval time.Duration, labels Labels, pc Prob
 
 	l := prometheus.Labels{
 		"name":  name,
-		"class": pc.Class,
+		"class": string(pc.Class),
+	}
+	if pc.Class == ClassDERPUDP {
+		if strings.Contains(name, "udp6") {
+			l["af"] = "udp6"
+		} else {
+			l["af"] = "udp"
+		}
```

Comment:
Take a look at `derpProber.ProbeUDP()` and the returned `ProbeClass.Labels`. This is where we can tack on additional labels.

Reply:
That's much nicer, thanks!